src/brevitas/graph/gpfq.py

```diff
@@ -302,13 +308,36 @@ def __init__(
             p=p)
         self.accumulator_bit_width = accumulator_bit_width
         assert self.accumulator_bit_width is not None
-        self.requires_quant_input = True  # force true
+
+    def process_input(self, inp):
+        inp = super().process_input(inp)
+        inp = self.layer.input_quant(inp)
```

Comment:
Is this here because we attach forward-pre-hook to the layer?

Reply:
Exactly. So the quant layer input is a tensor, but the input to the inner forward pass is after the input quant. I added a unit test to make sure that works as well since it wasn't before.